app/code/Magento/Eav/Model/Config.php

```diff
@@ -461,16 +471,16 @@ public function getEntityType($code)
      * @param string $entityType
      * @return $this
      */
-    protected function _initAttributes($entityType)
+    protected function _initAttributes($entityType, $storeId)
```

Comment:
I think we should add some default values, like "global", to make this change backward compatible.
```suggestion
    protected function _initAttributes($entityType, $storeId = 0)
```

Or make it nullable and try to retrieve the current store in such case.
```suggestion
    protected function _initAttributes($entityType, $storeId = null)
```
What do you think?

Reply:
I am absolutely sure that we need default value for the new argument but not absolutely sure which variant is better.
Let's retrieve the current store.